enterprise/api/v1/views/enterprise_group.py

```diff
@@ -162,6 +162,13 @@ def get_learners(self, request, *args, **kwargs):
                                                     pending_users_only=pending_users_only,
                                                     fetch_removed=show_removed)
 
+            if learners := param_serializers.validated_data.get('learners'):
```

Comment:
[question] since the `learners` field is required in the serializer, do we still need this conditional block?

Reply:
it's not required, emails are required if you provide the `learners` param.